wps-cloudformation-template.yaml

```diff
@@ -137,7 +140,7 @@ Mappings:
       request: "request.xml"
       templates: "templates.xml"
       bootstrapCss: "bootstrap.min.css"
-      aodnCss: "AODNTheme.css"
+      jobStatusCss: "AODNStatusPage.css"
```

Comment:
This will require us to modify jenkins to provide this as an input - pity it can't be served from the deployed artifact.  Something to llok at with logos etc this iteration perhaps

Reply:
Sorry - because it was a separate CSS i renamed to indicate what it was styling